# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/udev/hotfix-or1k.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- systemd-258/src/include/override/sys/syscall.h.vanilla	2025-10-02 11:28:32.454066025 +0200
+++ systemd-258/src/include/override/sys/syscall.h	2025-10-02 11:30:43.686684225 +0200
@@ -295,6 +295,8 @@
 #    define systemd_NR_open_tree_attr 467
 #  elif defined(__hppa__)
 #    define systemd_NR_open_tree_attr 467
+#  elif defined(__or1k__)
+#    define systemd_NR_open_tree_attr 467
 #  elif defined(__powerpc__)
 #    define systemd_NR_open_tree_attr 467
 #  elif defined(__riscv)
@@ -507,6 +509,8 @@
 #    define systemd_NR_removexattrat 466
 #  elif defined(__hppa__)
 #    define systemd_NR_removexattrat 466
+#  elif defined(__or1k__)
+#    define systemd_NR_removexattrat 466
 #  elif defined(__powerpc__)
 #    define systemd_NR_removexattrat 466
 #  elif defined(__riscv)
@@ -579,6 +583,8 @@
 #    define systemd_NR_setxattrat 463
 #  elif defined(__hppa__)
 #    define systemd_NR_setxattrat 463
+#  elif defined(__or1k__)
+#    define systemd_NR_setxattrat 463
 #  elif defined(__powerpc__)
 #    define systemd_NR_setxattrat 463
 #  elif defined(__riscv)
